app/helpers/submission_inputs_helper.rb

```diff
@@ -444,7 +444,7 @@ def generate_list_text_input(attr, helper_text: nil, long_text: false)
 
     generate_list_field_input(attr, name, label, values, helper_text: helper_text) do |value, row_name, id|
       if long_text
-        text_area_tag(row_name, value, class: 'input-field-component', label: '')
+        text_area_tag(row_name, value, class: 'input-field-component', label: '', resize: true)
```

Comment:
does this work ? 

`text_area_tag` is not our component `text_area_input`

Show a screenshot of the notes inputs bellow the description.

Reply:
No it doesn't work, I didn't notice that it's another component, done now:
<img width="752" alt="Capture d’écran 2024-05-31 à 11 00 48" src="https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/61744974/5a0ef496-1f52-465e-ad2a-ce2e368bd508">